### DISCUSSION TITLE
Shared paper collection

### DISCUSSION THREAD

**Discussion by tmnku** [Future-Scholars/paperlib#523] [Q&A] [answered]
Is it possible to have a shared collection of papers so multiple users with individual database accounts can cooperate? I was trying to do this but when I change the database user inside paperlib I can only see the papers that this user added. However when logging into MongoDB's website I can see all papers from both users residing in the same database.

**Comment by GeoffreyChen777** [comment, marked as answer]
Currently, one user can only see his/her own papers in the online database. There is no feature for shared collection yet.

**Reply by tmnku** [reply]
Thanks for your answer. Is this feature something that is planned for the future/ something that would be of interest to the project? I would have a look into it then.

**Reply by GeoffreyChen777** [reply]
Hi you can open an issue for this. If many user wants this feature, I will consider it.
